packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -50,6 +50,7 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
         // See http://crbug.com/715363
         // We use this flag to work-around this issue.
         '--disable-dev-shm-usage',
+        '--force-gpu-mem-discardable-limit-mb',
```

Comment:
This also seems to help in Brittany's PR, #10095

Reply:
I wanted to check what makes the difference, flags or disabled tests/other fixes.
Looks like this flag alone doesn't do much: https://github.com/google/web-stories-wp/runs/4774876792?check_suite_focus=true
I'll try the second.

Reply:
Locally I removed both flags from #10095 and it's still ✅ 
![image](https://user-images.githubusercontent.com/21036224/148945408-2e28fbad-ebc2-4078-81d3-b044c55bfa86.png)
I will just check it on CI and then look into disabled tests/other fixes.

Reply:
Running tests locally hasn't been an issue AFAIK, only running on CI

Reply:
I've had a few buffer/memory-related messages locally, like this one from the latest CI run (with those 2 flags):

![image](https://user-images.githubusercontent.com/21036224/148959017-df101aac-ee83-41a5-bf8c-140386e069c9.png)

So far it looks like the flags are not an instant fix, but they might lower the overall amount of used memory/buffer size produced.